### PR TITLE
lrcalc: init at 1.2

### PR DESCRIPTION
**Pull request by timokau**
###### Motivation for this change

Package lrcalc.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by dotlambda** [comment]
@GrahamcOfBorg build lrcalc

File: pkgs/applications/science/math/lrcalc/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      sha256 = "02kaqx5s3l642rhh28kn2wg9wr098vzpknxyl4pv627lqa3lv9vm";
+    })
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Littlewood-Richardson calculator";
+    homepage = http://math.rutgers.edu/~asbuch/lrcalc/;
+    license = licenses.gpl2Plus;
+    maintainers = with maintainers; [ timokau ];
+    platforms = platforms.all;
```
Comment (dotlambda) [inline pkgs/applications/science/math/lrcalc/default.nix]: Tested?

File: pkgs/applications/science/math/lrcalc/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+, fetchpatch
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.2";
+  pname = "lrcalc";
+  name = "${pname}-${version}";
+
+  src = fetchFromBitbucket {
```
Comment (dotlambda) [inline pkgs/applications/science/math/lrcalc/default.nix]: Why not use `fetchurl` with http://sites.math.rutgers.edu/~asbuch/lrcalc/lrcalc-1.2.tar.gz?

Comment (timokau) [reply pkgs/applications/science/math/lrcalc/default.nix]: I thought `fetchFromSource` was generally preferred, is it not? Its closer to the source, less likely to get out of date and guaranteed to actually contain the correct source.

Comment (timokau) [reply pkgs/applications/science/math/lrcalc/default.nix]: I personally prefer it this way (`fetchFromVCS`).

**Comment by dotlambda** [comment]
@GrahamcOfBorg build lrcalc

**Pull request merged**